code/datums/gamemode/role/role.dm

```diff
@@ -145,6 +145,8 @@
 		return 0
 
 	antag = M
+	antag.key = M.current.client.ckey
+	antag.active = TRUE
```

Comment:
Not the place for that

Reply:
This is the only line of code in this PR that actually fixed anything, but it is indeed not the right way to fix it